filebeat/module/haproxy/log/_meta/fields.yml

```diff
@@ -14,6 +14,7 @@
     - name: captured_headers
       description: >
         List of headers captured in the response due to the presence of the "capture response header" statement in the frontend.
+      # Do we need text?
```

Comment:
Only one example of a captured header, and it looks like this is only the header name, no value. So agree to move to keyword

Reply:
https://github.com/elastic/beats/pull/10397